### ISSUE TITLE
Chromatic error loading S3_*_Table_Save.txt format files output from Eureka!

### ISSUE THREAD

**Issue by meganmansfield** [ers-transit/ers-data-checkpoint-showcase#1]
When trying to load 1D spectra from the 'S3_*_Table_Save.txt' files output by the Eureka! pipeline, I'm encountering an error in the Chromatic readers/eureka.py routine. The error is being thrown because it's looking for the keyword data['bjdtdb'], which apparently doesn't exist. I'm guessing the naming convention in Eureka! has changed since this routine was written. I believe the keyword is just called 'time' now. (Sorry on behalf of the Eureka team)

**Comment by zkbt** [comment]
Thanks, @meganmansfield ! I just updated v0.0.10 to now look for `time` first, and then `bjdtdb` second. Can you try it now to see if it works?

**Comment by meganmansfield** [comment]
Haha, that part worked, but it's now throwing another error:
<img width="1015" alt="Screen Shot 2022-03-10 at 1 06 33 PM" src="https://user-images.githubusercontent.com/32969116/157746179-e0dcaabf-43de-4520-90e4-2526a1f283b2.png">
Close though!

**Comment by zkbt** [comment]
Ha, so close! Would you mind sending me the file you're working with? It'd be nice to have access to all the snazzy diagnostics you put into those files.

I also noticed that my new simple text reader got the axes swapped for the `simplespec.txt` file you submitted. I'll take a look at that.

**Comment by meganmansfield** [comment]
Yeah, no problem. I'm having a hard time attaching the Eureka file here, but I'll send it in Slack.

**Comment by zkbt** [comment]
@meganmansfield , I think I finally fixed this. It should be good now in versions >0.0.16. It was a hilariously tiny typo.

**Comment by zkbt** [comment]
Could you please try reading in one of your S3 files to make sure my fix actually works?

**Comment by meganmansfield** [comment]
Sorry, just got through checking my emails after the rush to get some HST proposals in after the data challenge ended. But my S3 files are properly loading in now!
